### DISCUSSION TITLE
Erro sd

### DISCUSSION THREAD

**Discussion by romehi** [sanni/cartreader#198] [Need help]
Bom dia tudo bem?

Acabei de terminar a montagem seguindo todas as etapas.
Porém ainda não coloquei o cartão de memória mas ao ligar o carrinho já dá um erro de sd o que pode ser?
![IMG_20210720_111757966](https://user-images.githubusercontent.com/84623440/126340480-78fac788-c0e6-4580-9262-417b5cabdaa9.jpg)

**Comment by Dakkaron** [comment]
As far as Google Translate translated correctly (btw, it might help to post in English) you did not put the SD card in.

`SD Error` appears if:

- There is no SD card inserted
- The SD card is formatted the wrong way (see the wiki to find out how to format it correctly)
- The connections aren't soldered correctly (check with a multimeter that all connections to the SD card are correct, especially check that the 5V-pin has actually 5V on it)

**Comment by sanni** [comment]
The SDFat v2 library seems to have some compatibility issues with slow SD cards. Can you try another SD card? Also exFAT seems to work better than FAT32 for some SD cards.
